From: Petr Tesarik <ptesarik@suse.cz>
Subject: kABI: protect struct struct smc_buf_desc
Patch-mainline: never, kabi
References: bnc#1117947, LTC#173662

Upstream commit c7674c001b1143a5bc6b36efc7adc4bdd8ff5e76 renames
field reused to wr_reg, giving it a slightly different semantics.
However, it is very unlikely that a 3rd party module would need
the information at all. And even if it did, it should not care
about the difference, as it is essentially a matter of timing.

Signed-off-by: Petr Tesarik <ptesarik@suse.cz>
---
 net/smc/smc_core.h |    4 ++++
 1 file changed, 4 insertions(+)

--- a/net/smc/smc_core.h
+++ b/net/smc/smc_core.h
@@ -129,7 +129,11 @@ struct smc_buf_desc {
 	struct page		*pages;
 	int			len;		/* length of buffer */
 	u32			used;		/* currently used / unused */
+#ifdef __GENKSYMS__
+	u8			reused	: 1;	/* new created / reused */
+#else
 	u8			wr_reg	: 1;	/* mem region registered */
+#endif
 	u8			regerr	: 1;	/* err during registration */
 	union {
 		struct { /* SMC-R */
